packages/cactus-plugin-ubiquity/src/main/typescript/generated/openapi/typescript-axios/.gitignore

```diff
@@ -0,0 +1,4 @@
+wwwroot/*.js
+node_modules
+typings
+dist
```

Comment:
@RafaelAPB Please delete this file - it is ignored in the open api generator ignore file as well.